api/v1alpha1/nodepool_types.go

```diff
@@ -314,6 +316,21 @@ type NodePoolPlatform struct {
 type KubevirtNodePoolPlatform struct {
 	// NodeTemplate Spec contains the VirtualMachineInstance specification.
 	NodeTemplate *capikubevirt.VirtualMachineTemplateSpec `json:"nodeTemplate,omitempty"`
+
+	// RootVolume specifies configuration for the root volume of node instances.
+	//
+	// +optional
+	RootVolume *KubeVirtRootVolume `json:"rootVolume,omitempty"`
```

Comment:
can we get away with not adding this RootVolume info to the nodepool?

Would it be possible to just have the size and storage class as cmd options that render the NodeTemplate with the expected size/storage class on the datavolumetemplate?

Reply:
weird this exact same comment got posted twice. Not sure how that happened.